libs/components/src/lib/text-field/text-field.scss

```
@@ -52,14 +53,18 @@ $low-ink-color: --_low-ink-color;
 
 .label:not(slot),
 .label::slotted(label) {
-	color: var(#{constants.$vvd-color-canvas-text});
-	contain: inline-size;
-	font: var(#{constants.$vvd-typography-base});
-	grid-column: 1 / 4;
-	grid-row: 1;
+	/* As label is in light dom, use !important and initial to prevent values getting overridden by global CSS resets etc. */
+	box-sizing: initial !important;
+	padding: initial !important;
+	margin: initial !important;
+	color: var(#{constants.$vvd-color-canvas-text}) !important;
+	contain: inline-size !important;
+	font: var(#{constants.$vvd-typography-base}) !important;
+	grid-column: 1 / 4 !important;
+	grid-row: 1 !important;
```

```suggestion
	grid-column: 1 / 4;
	grid-row: 1;
```
I don't think this is needed - why would some reset or css variable setting values for grid?